pkg/httputils/httputils_test.go

```diff
@@ -0,0 +1,59 @@
+package httputils
+
+import (
+	"testing"
+)
+
+func TestDownload(t *testing.T) {
+	_, err := Download("http://docker.com")
```

Comment:
@Triara I think you could do ``if _, err := …; err !=nil {` too (I tend to prefer that notation but it's up to you).